packages/core/src/Stores/client-store.js

```diff
@@ -1720,10 +1723,11 @@ export default class ClientStore extends BaseStore {
             }
 
             if (this.account_settings) this.setPreferredLanguage(this.account_settings.preferred_language);
-            this.loginid !== 'null' && RudderStack.setAccountType(this.loginid.substring(0, 2));
+            this.loginid !== 'null' && Analytics.setAttributes({ account_type: this.loginid.substring(0, 2) });
```

Comment:
already set

Reply:
I think this is fine and also in the other places as well, the values are getting changed so it should be updated for our rudderstack instance

Reply:
for this I found it valid, thanks for explanations